### PR TITLE
This change can be introduced after pull/88

### PR DESCRIPTION
**Pull request by tsheasha**
Since we can add dsimensions to metrics as of pull/88 we can add this
collector easily and make it work as intended. The value of 1 for this
metric is intended as yo make it possible to sum all the timeseries
produced by the metric name and be able to get a count for each
os_distro in a distributed system.

### CODE REVIEW

File: src/diamond/collectors/osdistro/test/testosdistro.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            'subprocess.Popen.communicate',
+            Mock(return_value=(
+                self.getFixture('ubuntu').getvalue(),
+                '')))
+
+        patch_communicate.start()
+        self.collector.collect()
+        patch_communicate.stop()
+
+        self.assertPublishedMany(publish_mock, {
```
Comment (tiras-j) [inline src/diamond/collectors/osdistro/test/testosdistro.py]: Do we have a way to assert on dimensions as well? Would be convenient to verify that they are being properly sent, especially for a collector like this where the dimensions are the only vital information. If you can call this a collector...

Comment (zachm) [inline src/diamond/collectors/osdistro/test/testosdistro.py]: +1

**Comment by zachm** [comment]
Other than @tiras-j's comment I am good with this. Fix and ship. One more!

File: src/diamond/collectors/osdistro/test/testosdistro.py (hunk 1 above)
Comment (tsheasha) [inline src/diamond/collectors/osdistro/test/testosdistro.py]: indeed that is true, I was planning to do it but it got lost in the sands of time.

**Comment by tiras-j** [comment]
Awesome! Shipit!

**Comment by poros** [comment]
+1

**Pull request merged**